apps/src/code-studio/pd/application/teacher/FindYourRegion.jsx

```diff
@@ -0,0 +1,251 @@
+import {LabelsContext} from '../../form_components_func/LabeledFormComponent';
+import {LabeledRadioButtons} from '../../form_components_func/labeled/LabeledRadioButtons';
+import {
+  FormContext,
+  getValidationState
+} from '../../form_components_func/FormComponent';
+import {isZipCode} from '@cdo/apps/util/formatValidation';
+import {useRegionalPartner} from '../../components/useRegionalPartner';
+import SchoolAutocompleteDropdown from '@cdo/apps/templates/SchoolAutocompleteDropdown';
+
+const PD_RESOURCES_URL =
+  'https://support.code.org/hc/en-us/articles/115003865532';
+const CS_TEACHERS_URL = 'https://code.org/educate/community';
+const INTERNATIONAL = 'Other country';
+const US = 'United States';
+
+const FindYourRegion = props => {
+  const {onChange, errors, data} = props;
+  const resetCountry = () => onChange({country: US});
+  const exitApplication = () => (window.location = PD_RESOURCES_URL);
+  const [regionalPartner] = useRegionalPartner(data);
+  useEffect(() => {
+    onChange({
+      regionalPartnerId: regionalPartner?.id,
+      regionalPartnerGroup: regionalPartner?.group,
+      regionalPartnerWorkshopIds: (regionalPartner?.workshops || []).map(
+        workshop => workshop.id
+      )
+    });
+  }, [regionalPartner, onChange]);
+
+  const getProgramInfo = program => {
+    switch (program) {
+      case PROGRAM_CSD:
+        return {name: 'CS Discoveries', shortName: 'CSD'};
+      case PROGRAM_CSP:
+        return {name: 'CS Principles', shortName: 'CSP'};
+      case PROGRAM_CSA:
+        return {name: 'CSA', shortName: 'CSA'};
+      default:
+        return {name: 'CS Program', shortName: null};
+    }
+  };
+  const programInfo = getProgramInfo(data.program);
+  const isOffered = regionalPartner?.pl_programs_offered?.includes(
+    programInfo.shortName
+  );
+
+  const renderInternationalModal = () => {
+    return (
+      <Modal show={data.country === INTERNATIONAL}>
+        <Modal.Header>
+          <Modal.Title>
+            Thank you for your interest in Code.org’s Professional Learning
+            Program.
+          </Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          At this time, we are only able to provide this program to teachers in
+          the United States. Please visit our website for additional Code.org{' '}
+          <a href={PD_RESOURCES_URL} target="_blank" rel="noopener noreferrer">
+            professional development resources
+          </a>{' '}
+          and opportunities to connect with other{' '}
+          <a href={CS_TEACHERS_URL} target="_blank" rel="noopener noreferrer">
+            computer science teachers
+          </a>
+          .
+        </Modal.Body>
+        <Modal.Footer>
+          <Button onClick={resetCountry} bsStyle="primary">
+            Continue as United States Teacher
+          </Button>
+          <Button onClick={exitApplication}>Exit Application</Button>
```

Comment:
As `exitApplication` goes to a specific url, can you use the `href` prop here instead?

Reply:
Good call!